app/templates/agenda.html

```diff
@@ -25,28 +19,45 @@
     <div>
         <a class="btn agenda_button" href="/agenda?days=30">{{ gettext("Next month") }}</a>
     </div>
+    <div>
+        <button class="btn agenda_button graph" type="button" name="{{events_for_graph}}">{{ gettext("Week graph") }}</button>
+    </div>
 </div>
 
 
 <div class="container mt-4">
     {% if start_date > end_date %}
-        <p>{{ gettext("Start date is greater than end date")  }}</p>
+    <p>{{ gettext("Start date is greater than end date") }}</p>
     {% elif events | length == 0 %}
-        <p>{{ gettext("No events found...")  }}</p>
+    <p>{{ gettext("No events found...") }}</p>
     {% elif start_date == end_date %}
-        <h1>{{ start_date.strftime("%d/%m/%Y") }}</h1>
+    <h1>{{ start_date.strftime("%d/%m/%Y") }}</h1>
     {% else %}
     <h1>{{ start_date.strftime("%d/%m/%Y") }} - {{end_date.strftime("%d/%m/%Y") }}</h1>
     {% endif %}
 </div>
 
-<div>
-    {% for events_date, events_list in events.items() %}
-    <div class="shown_dates">{{ events_date.strftime("%d/%m/%Y") }}</div>
+<div class="row">
+    <div class="col-5">
+        {% for events_date, events_list in events.items() %}
+        <div class="shown_dates">{{ events_date.strftime("%d/%m/%Y") }}</div>
         {% for event in events_list %}
         <div class="event_line"> {{ event[0].start.time().strftime("%H:%M") }} - <a href="/event/view/{{ event[0].id }}">{{ event[0].title }}</a> , duration: {{ event[1] }}</div>
-        {% endfor %}
-    {% endfor %}
+        {% endfor %} {% endfor %}
+    </div>
+
+
+    <div class="col-4">
+        <div class="card-body">
+            <canvas id="myChart" width="400" height="400"></canvas>
+            <script src="../static/graph.js"></script>
```

Comment:
Prefer loading all the `script`s in the end of the `body` tag

Reply:
done!
and also changed to:    
<script src="{{ url_for('static', path='/graph.js') }}"></script>